pagespeed/kernel/http/response_headers.cc

```diff
@@ -321,23 +321,28 @@ bool ResponseHeaders::CombineContentTypes(const StringPiece& orig,
 }
 
 bool ResponseHeaders::MergeContentType(const StringPiece& content_type) {
-  bool ret = false;
+  bool ret = std::find_if(
+      content_type.begin(), content_type.end(), iscntrl) == content_type.end();
```

Comment:
I have to admit to a certain paranoia about the likes of iscntrl, due to their locale-sensitivity.
Maybe our inline bool IsNonControlAscii(char c) from string_util.h would be an acceptable substitute?